### PR TITLE
Create 2022-04-22-learning-vs-negative-thoughts-en.md

### PR DESCRIPTION
**Pull request by AlmDea**

### CODE REVIEW

**Comment by NatalieKonstantinova** [comment]
@mikhaylov-ya погляди пжл

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+3. **Being open.** Exclude one of the main conditions of mental block.
+4. **Progress**. Accept the realities of the learning process.
+ 
+## Step 1: Realization
+
+First of all, it’s necessary to notice and realize the emerging resistance of the mind. Treat your body and mind as a scientific experiment: they give you, the observer, some signals, then they react to actions and environment.
+
+image: "https://guides.hexlet.io/assets/images/learning/learning_1.png"
+
+Try to notice or even write down the conditions under which such thoughts typically occur. A lot of factors affect the mental state, including physical condition, work environment, sleep, nutrition, stress, goals, relationships, etc.
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:26]: factors => things

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Keeping a journal can be a good idea. Write everything about your everyday life: how much you sleep, what do you do, what do you eat, how much do you read, how do you relax and how are your trainings going.
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:28]: how much you sleep, what do you do, what do you eat, how much do you read, how do you relax and how are your trainings going =>
how much you sleep, what you do, what you eat, how much you read, how you relax, and how is your training going.

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+It will be great if you can identify the reasons of good and bad days. Sleeping less than 7 hours plus meeting someone before lunch is almost always correlated with academic failures. Or physical activity and reduced consumption of sweets has something with successful trainings. Perhaps there isn't just a correlation here but a causal link.
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:30]: correlated with academic failures => have something with learning problems

Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:30]: Perhaps this is not a mere correlation, but a causality.

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+Mindset is a collection of beliefs, ideas and concepts that held by oneself.
+
+**Fixed mindset** is characterized by such beliefs:
+
+- I am who I am and people don't change
+- success depends on talent which is innate
+- people are fixed into clear categories: techies, humanitarians, strong and weak, smart and stupid, etc.
+- I should avoid changes and challenges to prevent the possibility of failure
+- putting in effort is worthless
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:48]: efforts are worthless

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+[Carol Dweck](https://en.wikipedia.org/wiki/Carol_Dweck), Stanford University psychologist, identified two core mindsets: fixed mindset and growth mindset.
+
+image: "https://guides.hexlet.io/assets/images/learning/learning_2.png"
+
+Mindset is a collection of beliefs, ideas and concepts that held by oneself.
+
+**Fixed mindset** is characterized by such beliefs:
+
+- I am who I am and people don't change
+- success depends on talent which is innate
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:45]: success relies on innate talent

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+- people are fixed into clear categories: techies, humanitarians, strong and weak, smart and stupid, etc.
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:46]: are fixed into clear => fit into specific

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+- I should avoid changes and challenges to prevent the possibility of failure
+- putting in effort is worthless
+- I need to choose a path of least effort
+- being criticised is bad
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:50]: criticized

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+Mindset is a collection of beliefs, ideas and concepts that held by oneself.
+
+**Fixed mindset** is characterized by such beliefs:
+
+- I am who I am and people don't change
+- success depends on talent which is innate
+- people are fixed into clear categories: techies, humanitarians, strong and weak, smart and stupid, etc.
+- I should avoid changes and challenges to prevent the possibility of failure
+- putting in effort is worthless
+- I need to choose a path of least effort
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:49]: the least

File: _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+- effort leads to mastery
+- I believe failures are just temporary setbacks and learning opportunities
+- constructive feedback as a source of information
+- complaints make no sense and distract from the things that really matter
+
+You can notice a recursive trap: fixed mindset makes a person think that his fate is predetermined and he won’t be able to change.
+
+Studies of mindsets lead to an interesting conclusion: the quality of life and satisfaction depend more on a subjective attitude, and *not on talent or intelligence.* In other words, a person is able to develop a growth mindset with beliefs and rational evidence.
+
+Use your emotional resistance ("probably, this is not for me", "it's obvious that I'm not a tech guy...", "I’d never make it as a programmer") as a starting point in your mindset transformation.
```
Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:68]: this is not for me => this is not my thing

Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:68]: this is not for me => this is not my thing

Comment (mikhaylov-ya) [inline _i18n/en/_posts/2022-04-22-learning-vs-negative-thoughts-en.md:68]: I'd => I would

**Pull request merged**